src/main/java/info/skyblond/nekohit/neo/domain/Messages.java

```diff
@@ -0,0 +1,87 @@
+package info.skyblond.nekohit.neo.domain;
+
+public class Messages {
```

Comment:
I would call the class **ExceptionMessages**, **ErrorMessages** or something more explicit. **Messages** seems a bit too generic for imho.

Reply:
Renamed in 727ae5a